guides/get-started-microservices-on-kubernetes/simple-microservice-client-part2.adoc

```diff
@@ -0,0 +1,293 @@
+
+== Microservice B - the server
+
+=== Image Registry
+
+To make the `{my-jaxrs-app-docker-image-name-server}:latest` Docker Image available to Kubernetes, you need to push it to some Image Registry that is accessible by the Kubernetes cluster you want to use.
+
+==== Quay.io
+
+There are many Image Registries you can use: in this guide, we will push the `{my-jaxrs-app-docker-image-name-server}:latest` Docker Image, to the link:https://quay.io[quay.io, window="_blank"] Image Registry.
+
+Create a public repository named `{my-jaxrs-app-docker-image-name-server}` on link:https://quay.io[quay.io, window="_blank"] (e.g. link:https://quay.io/repository/{quay-io-account-name}/my-jaxrs-app[https://quay.io/repository/{quay-io-account-name}/{my-jaxrs-app-docker-image-name-server}, window="_blank"]).
+
+NOTE: replace `{quay-io-account-name}` with the name of your account in all the commands that will follow
+
+Tag the Docker image:
+
+[source,bash,subs="normal"]
+----
+podman tag {my-jaxrs-app-docker-image-name-server} quay.io/{quay-io-account-name}/{my-jaxrs-app-docker-image-name-server}
+----
+
+Push the `{my-jaxrs-app-docker-image-name-server}` Docker Image to it:
+
+[source,bash,subs="normal"]
+----
+podman push quay.io/{quay-io-account-name}/{my-jaxrs-app-docker-image-name-server}
+----
+
+At this point, the `{my-jaxrs-app-docker-image-name-server}:latest` Docker Image should be publicly available and free to be consumed by any Kubernetes Cluster;
+
+=== Deploy on Kubernetes
+
+Create a file named `{my-jaxrs-app-docker-image-name-server}-deployment.yaml`:
+
+.{my-jaxrs-app-docker-image-name-server}-deployment.yaml:
+[source,yaml,subs="normal"]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {my-jaxrs-app-docker-image-name-server}-deployment
+  labels:
+    app: {my-jaxrs-app-docker-image-name-server}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {my-jaxrs-app-docker-image-name-server}
+  template:
+    metadata:
+      labels:
+        app: {my-jaxrs-app-docker-image-name-server}
+    spec:
+      containers:
+        - name: {my-jaxrs-app-docker-image-name-server}
+          image: quay.io/tborgato/{my-jaxrs-app-docker-image-name-server}
+          ports:
+            - containerPort: 8080
+            - containerPort: 9990
+          livenessProbe:
+            httpGet:
+              path: /health/live
+              port: 9990
+          readinessProbe:
+            httpGet:
+              path: /health/ready
+              port: 9990
+          startupProbe:
+            httpGet:
+              path: /health/started
+              port: 9990
+----
+
+Deploy to your Kubernetes Cluster:
+
+[source,bash,subs="normal"]
+----
+kubectl apply -f {my-jaxrs-app-docker-image-name-server}-deployment.yaml
+----
+
+=== Create Kubernetes ClusterIP Service
+
+We create a service to consume the services exposed by **{my-jaxrs-app-docker-image-name-server}** from inside Kubernetes;
```

Comment:
Again, I assume this needs to happen in the client, but it's not really described. Or I guess in any directory, but seems it should specify.

Reply:
I added a link to https://kubernetes.io/docs/concepts/services-networking/service/ which should clarify this statement